yupana/processor/report_slice_processor.py

```diff
@@ -186,6 +192,81 @@ def generate_upload_candidates(self):
                       for key in host.keys() if key not in ['cause', 'status_code']}
         return candidates
 
+    def _match_regex_and_find_version(self, os_release):
+        """Match Regex with os_release and return os_version."""
+        source_os_release = os_release.strip()
+        if not source_os_release:
+            return None
+
+        match_result = OS_RELEASE_PATTERN.match(source_os_release)
+        parsed_info = match_result.groupdict()
+        os_version = parsed_info['version'].strip()
+        LOG.info(
+            format_message(
+                self.prefix,
+                "os version after parsing os_release: '%s'"
+                % os_version,
+                account_number=self.account_number,
+                report_platform_id=self.report_platform_id))
+        return os_version
+
+    def _transform_os_release(self, host: dict):
+        """Transform 'system_profile.os_release' label."""
+        os_key_present = 'os_release' in host['system_profile']
+        if (not os_key_present or (os_key_present and not isinstance(
+                host['system_profile']['os_release'], str))):
+            return host
+
+        os_release = host['system_profile']['os_release']
```

Comment:
```suggestion
        os_release = host['system_profile'].get('os_release')
        if not isinstance(os_release, str):
            return host
```

`get` will not throw an exception, and `isinstance` will return false if the value is Nothing